app/javascript/src/components/Dashboard/Notes/NewNoteForm.jsx

```diff
@@ -1,69 +1,74 @@
 import React from "react";
 
 import { Formik, Form } from "formik";
-// import { Button } from "neetoui";
 import { Input, Textarea } from "neetoui/formik";
-import { Dropdown, Label, Toastr, Button } from "neetoui/v2";
+import { Toastr, Button } from "neetoui/v2";
+import { Select } from "neetoui/v2/formik";
 import * as yup from "yup";
 
-import { CONTACTS } from "constants/notes";
+import {
+  CONTACTS_DROPDOWN_VALUES,
+  TAGS_DROPDOWN_VALUES,
+} from "constants/notes";
```

Comment:
Let's keep the constants scoped to `Notes` folder.

Reply:
Do you mean that instead of `constants/notes` it should be `notes/constants`? And use it from there.

Reply:
Yup.